content/post/kubedb-v2024.06.04/index.md

```diff
@@ -0,0 +1,788 @@
+
+To issue a certificate, the following Custom Resource (CR) of cert-manager is used:
+
+**Issuer/ClusterIssuer**: Issuers and ClusterIssuers represent certificate authorities (CAs) that can generate signed certificates by honoring certificate signing requests. All cert-manager certificates require a referenced issuer in a ready condition to attempt to serve the request. You can learn more details [here](https://cert-manager.io/docs/concepts/issuer/).
+
+**Certificate**: The cert-manager has the concept of Certificates that define the desired `x509` certificate which will be renewed and kept up to date. You can learn more details [here](https://cert-manager.io/docs/usage/certificate/).
+
+Here’s a sample YAML for TLS-enabled Microsoft SQL Server:
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: MSSQLServer
+metadata:
+  name: mssql-standalone-tls
+  namespace: demo
+spec:
+  version: "2022-cu12"
+  replicas: 1
+  storageType: Durable
+  tls:
+    issuerRef:
+      name: mssqlserver-issuer
+      kind: Issuer
+      apiGroup: "cert-manager.io"
+    certificates:
+      - alias: client
+        subject:
+          organizations:
+            - kubedb
+        emailAddresses:
+          - abc@appscode.com
+      - alias: server
+        subject:
+          organizations:
+            - kubedb
+        emailAddresses:
+          - abc@appscode.com
+    clientTLS: true
+  storage:
+    storageClassName: "standard"
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+  deletionPolicy: WipeOut
+```
+The users must specify the `spec.tls.issuerRef` field. If user set `spec.tls.clientTLS`: true  then tls enabled SQL Server will be provisioned. 
+If `tls.clientTLS`: false is specified then tls will not be enabled for SQL Server but the Issuer will be used to configure tls enabled wal-g proxy-server which is required for SQL Server backup restore.
+KubeDB uses the issuer or clusterIssuer referenced in the `tls.issuerRef` field, and the certificate specs provided in `tls.certificate` to generate certificate secrets using Issuer/ClusterIssuers specification. These certificate secrets includes `ca.crt`, `tls.crt` and `tls.key` etc. and are used to configure Microsoft SQL Server
+
+
+## MongoDB
+### MongoDBArchiver Shard Support: 
+We are pleased to announce that this release includes support for the `MongoDBArchiver` in Sharded MongoDB Cluster environments. This significant enhancement enables Point-in-Time Recovery (PITR) for the Sharded MongoDB Cluster managed by KubeDB, providing the capability to restore data to any specific point in time following a disaster. This constitutes a major feature addition that will greatly benefit users by improving disaster recovery processes and minimizing potential data loss.
```

Comment:
Upload the shard-archiver yamls in
https://github.com/kubedb/archiver-demo/tree/master/mongodb. 

Give the reference here.